UrbanAirship.Android.ADM.nuspec

```diff
@@ -11,12 +11,12 @@
 
       <dependencies>
          <group targetFramework="MonoAndroid">
-            <dependency id="urbanairship.android.core" version="10.1.1"/>
+            <dependency id="urbanairship.android.core" version="12.2.0"/>
          </group>
       </dependencies>
    </metadata>
 
    <files>
-     <file src="src/AirshipBindings.Android.ADM/bin/release/AirshipBindings.ADM.dll" target="lib/MonoAndroid41" />
+     <file src="src/AirshipBindings.Android.ADM/bin/release/AirshipBindings.Android.ADM.dll" target="lib/MonoAndroid41" />
```

Comment:
MonoAndroid41 ?

Reply:
That's what the target has been. Maybe that was current when we created the bindings? It is consistent in all of the Android bindings projects.

Reply:
afaict it's the name of the folder our built dll's go, not a setting or a library we're pulling in